apps/explorer/src/ui/TransactionAddressSection.tsx

```diff
@@ -8,6 +8,8 @@ import { CoinBalance } from './CoinBalance';
 import { Heading } from './Heading';
 import { TransactionAddress } from './TransactionAddress';
 
+// TODO: Move this out of shared components and into components/transactions
```

Comment:
Oh it's fine, we don't have a super strong convention around this, but as the app is getting bigger I'm noticing more UI components that aren't really re-usable but are page-specific. At first that was fine because we were rewriting a lot of the app, but now that we've got a lot of UI components, I just want to keep a better separation of "fully and truly re-usable, anywhere in the app" and "component for a specific part of the page that may or may not be re-usable but probably only makes sense in a specific context"